src/store/actions/toasts/toastsActions.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  CustomToastType,
+  ToastsEnum
+} from 'store/slices/toast/toastSlice.types';
+import { getStore } from 'store/store';
+import { getUnixTimestamp } from 'utils';
+
+export const addCustomToast = (
+  customToasts: CustomToastType,
+  currentToastId?: string
+) => {
+  getStore().setState(({ toasts: state }) => {
+    const toastId =
+      currentToastId || `custom-toast-${state.customToasts.length + 1}`;
```

Comment:
add toast 0 --> custom-toast-1
add toast 1 --> custom-toast-2
delete toast 0
add toast --> custom-toast-2 => you will have duplicates of custom-toast-2 
solution state.customToasts --> get last toast index and increase it by one

Reply:
done